SPECS/openssl/openssl.spec

```diff
@@ -38,7 +40,6 @@ Group: System Environment/Libraries
 OpenSSL is a toolkit for supporting cryptography. The openssl-libs
 package contains the libraries that are used by various applications which
 support cryptographic algorithms and protocols.
-Requires: openssl = %{version}-%{release}
```

Comment:
I have never done this: how have you verified that this package is not needed for libs during runtime?

Reply:
I analyzed the contents of packages. Since libs require openssl and openssl requires libs, there's a circular dependency there. However, looking at the openssl package you can only see a binary there. The binary links against the libs, not the other way around. Therefore it looks like someone overspecified the requirements here. Openssl is still going to pull the libs, but libs won't pull in openssl binary if you don't need it.

Reply:
Got it, makes sense and thank you for doing that!

---
In reply to: [542597389](https://github.com/microsoft/CBL-Mariner/pull/403#discussion_r542597389) [](ancestors = 542597389)